packages/block-editor/src/components/iframe/index.js

```diff
@@ -354,7 +354,7 @@ function Iframe( {
 		// but calc( 100px / 2px ) is not.
 		iframeDocument.documentElement.style.setProperty(
 			'--wp-block-editor-iframe-zoom-out-scale',
-			scale === 'default'
+			scale === 'auto-scaled'
```

Comment:
Is this just a case of not wanting to set a scale as a percentage, but instead a certain width?

Reply:
Yeah, I'm not entirely how the computation works, it's computed using the actual width of the window and a maxWidth of 750 for the zoom-out area. 

I would be fine removing it and setting a percentage but I don't have too much context here.

Reply:
My point is: does it make sense to have a % API? Are we using that as third party somewhere? Would a width work there instead?

Reply:
I always saw the state as representing a percentage of zoom 👍

Reply:
Any way we can make this auto scaled private? Maybe a symbol?